packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -707,6 +731,29 @@ function IntroductoryOfferCallout( {
 	return <DiscountCallout>{ text }</DiscountCallout>;
 }
 
+function PartnerLogo( {
```

Comment:
_PartnerLogo_ component to render the partner logo and a callout message informing that it is included with the user's hosting plan.